packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/index.js

```diff
@@ -57,9 +63,9 @@ module.exports.assertions = {
     try {
       callself.makeTheCall();
     } catch (e) {
-      throw Error('Referencing exported functions should be possible', {
-        cause: e,
-      });
+      throw Error(
+        'Referencing exported functions should be possible. \n' + e.stack,
```

Comment:
I’m on the fence about committing investigative code. Your call.

Reply:
While node 16 supports passing cause, ava doesn't seem to. This was an attempt to replicate the default result of passing cause, not to keep the investigative code. Switching to e.message